docs/API.md

```diff
@@ -198,6 +200,29 @@ The definition above shows what the `props.intl` object will look like that's in
 - **`IntlConfig`:** The intl metadata passed as props into the parent `<IntlProvider>`.
 - **`IntlFormatters`:** The imperative formatting API described below.
 
+#### `createIntl`
+
+This allows you to create an `IntlShape` object without using `Provider`. This allows you to format things outside of React lifecycle while reusing the same `intl` object. For example:
+
+```tsx
+import {createIntl, createIntlCache, RawIntlProvider} from 'react-intl'
+
+// This is optional but highly recommended
+// since it prevents memory leak
+const cache = createIntlCache()
```

Comment:
What kind of memory leak is trying to prevent?! Do you have some example so we can evaluate the use case? Tx!

Reply:
This is primarily for SSR. Right now `intl-format-cache` (or any memoization lib like `fast-memoize` or `_.memoize) creates its own cache object within a closure so what we've seen over time is that doesn't get released.